### PR TITLE
Standardization

### PR DESCRIPTION
**Pull request by frlwolf**
Fixes headers removing copyright, identating with the project standar, renaming with the correct project title, etc... 
Idents all files equaly, using ours (CopyIsRight) AppCode configuration. 
Renames `GDGValueAdapter` to `GDGValueTransformer`, because we are not developing in Java.
Remove unused methods, fix type missuse warnings, rename some misspelling variables...
And finally remove `GDGEntityManager`'s database (look at `CIRDatabase+GoldDigger` for more information)

@pietrocaselani be my guest

### CODE REVIEW

**Comment by frlwolf** [comment]
I know, I know... **ALL FILES** are modified... I just couldn't help myself

File: Pod/Classes/GDGColumn.h [hunk 1]
```diff
@@ -3,33 +3,33 @@
...
+{
+	GDGColumnTypeText,
 	GDGColumnTypeInteger,
 	GDGColumnTypeFloat,
-	GDGColumnTypeText,
 	GDGColumnTypeBlob,
 	GDGColumnTypeDate,
 	GDGColumnTypeDouble,
 	GDGColumnTypeBoolean,
-	GDGColumnTypeNull
```
Comment (pietrocaselani) [inline Pod/Classes/GDGColumn.h:19]: Aqui é Null e não none.

File: Pod/Classes/GDGColumn.m [hunk 2]
```diff
@@ -3,47 +3,52 @@
...
 }
 
-@implementation GDGColumn
-
-- (instancetype)initWithName:(NSString*)name type:(GDGColumnType)type primaryKey:(BOOL)primaryKey notNull:(BOOL)notNull
+- (instancetype)initWithName:(NSString *)name type:(GDGColumnType)type primaryKey:(BOOL)primaryKey notNull:(BOOL)notNull
 {
-	if (self = [super init])
+	self = [super init];
+	if (self)
```
Comment (pietrocaselani) [inline Pod/Classes/GDGColumn.m:45]: Por que duas linhas?

File: Pod/Classes/GDGEntity.h [hunk 3]
```diff
@@ -13,21 +12,21 @@
...
 
 @interface GDGEntity : NSObject
 
 @property (assign, nonatomic) NSUInteger id;
-@property (readonly, nonatomic) GDGEntityManager<GDGEntityFillDelegate>* db;
+@property (readonly, nonatomic) GDGEntityManager <GDGEntityFillDelegate> *db;
 
-- (BOOL)save;
-- (BOOL)drop;
++ (instancetype)entity;
```
Comment (pietrocaselani) [inline Pod/Classes/GDGEntity.h:26]: Meio estranho `[GDGEntity entity];` não? Sei que não é Java, então nada de `newInstance`, mas se for pensar em `sharedManager` ou `defaultManager`, `newEntity` não parece tão ruim.

File: Pod/Classes/GDGEntityManager.h [hunk 4]
```diff
@@ -13,23 +12,14 @@
 @class GDGEntitySettings;
 @class GDGHasOneRelation, GDGHasManyRelation, GDGBelongsToRelation;
 @class GDGEntityQuery;
-@class GDGValueAdapter;
 @class GDGTableSource;
 @class CIRDatabase;
 @class GDGColumn;
 
 @interface GDGEntityManager : NSObject <GDGEntityFillDelegate>
```
Comment (pietrocaselani) [inline Pod/Classes/GDGEntityManager.h:19]: Já podia renomar para `GDGEntityMapper` não?

File: Pod/Classes/GDGEntity.h (hunk 3 above)
Comment (frlwolf) [inline Pod/Classes/GDGEntity.h:26]: [[NSArray array]](https://developer.apple.com/library/mac/documentation/Cocoa/Reference/Foundation/Classes/NSArray_Class/#//apple_ref/occ/clm/NSArray/array)
[[NSDictionary dictionary]](https://developer.apple.com/library/mac/documentation/Cocoa/Reference/Foundation/Classes/NSDictionary_Class/#//apple_ref/occ/clm/NSDictionary/dictionary)
[[NSSet set]](https://developer.apple.com/library/mac/documentation/Cocoa/Reference/Foundation/Classes/NSSet_Class/#//apple_ref/occ/clm/NSSet/set)
[[NSString string]](https://developer.apple.com/library/mac/documentation/Cocoa/Reference/Foundation/Classes/NSString_Class/#//apple_ref/occ/clm/NSString/string)

Just following the standards

File: Pod/Classes/GDGColumn.m (hunk 2 above)
Comment (frlwolf) [inline Pod/Classes/GDGColumn.m:45]: Dunno O.O

File: Pod/Classes/GDGEntityManager.h (hunk 4 above)
Comment (frlwolf) [inline Pod/Classes/GDGEntityManager.h:19]: Will let that for the next PR

**Pull request merged**